### PR TITLE
Create ro-RO.json

### PR DESCRIPTION
**Pull request by haxxorer**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] My pull request is unique and no other pull requests have been opened for these changes
- [ ] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [ ] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Review by Arikatsu — COMMENTED** [review]
pog

**Comment by Secretboy-SMR** [comment]
;

File: src/main/resources/languages/ro-RO.json [hunk 1]
```diff
@@ -0,0 +1,388 @@
+   Preview
```
Comment (Birdulon) [inline src/main/resources/languages/ro-RO.json]:
```suggestion

```

File: src/main/resources/languages/ro-RO.json [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+      "title_avatars": "Avataruri",
+      "title_items": "Elemente",
+      "title_scenes": "Scene",
+      "title_monsters": "Monștri",
+      "header_id": "Id",
+      "header_command": "Comandă",
+      "header_description": "Descriere",
+      "header_avatar": "Avatar",
+      "header_item": "Element",
+      "header_scene": "Scenă";
```
Comment (Birdulon) [inline src/main/resources/languages/ro-RO.json]:
```suggestion
      "header_scene": "Scenă",
```

**Review by Birdulon — CHANGES_REQUESTED** [review]

**Review by Birdulon — APPROVED** [review]

**Pull request merged**